projects/dxc-ngx-cdk/src/lib/dxc-crud-table/dxc-crud-table.component.ts

```diff
@@ -984,8 +985,8 @@ export class DxcCrudTableComponent implements OnInit, ControlValueAccessor, OnCh
       };
     }
     this.formControlUpdater.emit(dataToEmit);
-
     this.suppgridForm = this.claimsForm;
+    this.parentForm.markAsDirty();
```

Comment:
same thing u already did in expand function so why its needed here ?

Reply:
when we click on add, form should be in edit mode without making any changes.